source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -101,7 +101,7 @@ def _check_train_cancel_disruption(dis):
             is_valid_disruption(dis, chaos_disrup=False)
             assert dis['contributor'] == rt_topic
             assert dis['disruption_id'] == 'disruption_bob'
-            assert dis['severity']['effect'] == 'NO_SERVICE'
+            assert dis['severity']['effect'] == 'no_service'
```

Comment:
You are changing the interface in a non retro compatible way, it's not possible.